### PR TITLE
consistent redirection after save

### PR DESCRIPTION
**Pull request by tejaswinichile**
Issue #422

### CODE REVIEW

**Comment by tejaswinichile** [comment]
@breezyboa _a Please check.

**Comment by tejaswinichile** [comment]
@breezyboa _a Please have a look.

File: app/javascript/Admin/Main.elm [hunk 1]
```diff
@@ -79,6 +78,12 @@ type PageState
     | TransitioningFrom Page
 
 
+type alias FlashElement =
```
Comment (breezyboa) [inline app/javascript/Admin/Main.elm]: Can we call this NotificationElement as it sounds more apt

File: app/javascript/Admin/Main.elm [hunk 2]
```diff
@@ -117,6 +126,7 @@ init flags location =
 
 type Msg
     = NavigateTo Route.Route
+    | DeleteFlashElement
```
Comment (breezyboa) [inline app/javascript/Admin/Main.elm]: rename this to `RemoveNotification`

File: app/javascript/Admin/Page/Feedback/List.elm [hunk 3]
```diff
@@ -82,39 +82,58 @@ view model =
...
+                    [ text "Closed Feedback" ]
+                ]
             ]
-            [ text "Closed Feedback" ]
-        , button
-            [ onClick (FeedbackListReloaded "open")
-            , class "btn btn-primary"
+        , div [ class "tab-content" ]
+            [ div [ class "tab-pane active form-group", id "open-feedbacks" ]
+                [ h3 [] [ text "Open Feedbacks: " ]
```
Comment (breezyboa) [inline app/javascript/Admin/Page/Feedback/List.elm]: Redundant h3 tag. remove it. the selected tab should be enough to convey to the user about whats being selected.

File: app/javascript/Admin/Page/Feedback/List.elm [hunk 4]
```diff
@@ -82,39 +82,58 @@ view model =
...
+                , div [ id "content-wrapper" ]
+                    (List.map
+                        (\feedback ->
+                            row model feedback
+                        )
+                        model.feedbackList
+                    )
+                ]
+            , div [ class "tab-pane form-group", id "closed-feedbacks" ]
+                [ h3 [] [ text "Closed Feedbacks: " ]
```
Comment (breezyboa) [inline app/javascript/Admin/Page/Feedback/List.elm]: Redundant h3 tag. Please remove it.

**Comment by tejaswinichile** [comment]
@breezyboa _a I have made changes according to PR review comments, please check.

**Pull request merged**